scripts/installscripts/tests/run_installation_spotify.sh

```diff
@@ -33,8 +33,8 @@ n
 y
 
 y
-myuser
-mypassword
+abcdefghijklmnopqrstuvwxyzßäöüÄÖÜ1234567890
+!\"§$%&/()=?´\`²³{[]}\+*~#',;.:-_ @€^°
```

Comment:
Should you have include `$` here?

Reply:
Hey, it's included after the paragraph sign. 
But it made no problems during my tests.

Reply:
Oh yes, so it is. Apologies, I missed it. But I am bit surprised it works.

Reply:
It just works cause of is not followed by an alpha numeric char... That's the flaw. 
I'm on it.